src/renderer/drawers/NFTViewerDrawer/index.js

```diff
@@ -169,8 +168,13 @@ export function NFTViewerDrawer({
               fontSize={7}
               lineHeight="29px"
               color="palette.text.shade100"
+              style={{
+                WebkitLineClamp: 3,
+                WebkitBoxOrient: "vertical",
+                overflow: "hidden",
+                display: "-webkit-box",
+              }}
```

Comment:
Probably need to add `text-overflow: ellipsis;` to avoid cuting the word too abruptly.

Reply:
Does it do anything in this case? It seems that the line clamp already inserts an ellipsis where it's needed in order to avoid overflow.

Reply:
You're right, my bad, looks like I have been mixing up 2 different solutions all along... 🤦